packages/server/postgres/getPgConfig.ts

```diff
@@ -1,12 +1,33 @@
+import {readFileSync} from 'fs'
+import path from 'path'
 import PROD from '../PROD'
 
-const getPgConfig = () => ({
-  user: process.env.POSTGRES_USER,
-  password: process.env.POSTGRES_PASSWORD,
-  database: process.env.POSTGRES_DB,
-  host: process.env.POSTGRES_HOST,
-  port: Number(process.env.POSTGRES_PORT),
-  max: Number(process.env.POSTGRES_POOL_SIZE) || PROD ? 20 : 5
-})
+const getSSL = () => {
+  try {
+    // optional env var, likely outside the app dir
+    const PG_CERT_DIR = process.env.PG_CERT_DIR!
+    const ca = readFileSync(path.join(PG_CERT_DIR, 'root.crt'))
+    const key = readFileSync(path.join(PG_CERT_DIR, 'postgresql.key'))
+    const cert = readFileSync(path.join(PG_CERT_DIR, 'postgresql.crt'))
+    return {ca, key, cert, rejectUnauthorized: false}
```

Comment:
I changed `rejectUnauthorized: true` to `false` because the certificate provided by GCP CloudSQL only has a as name of the server the one used by the CloudSQL Proxy (we are going to use it in the future). You can see other similar cases on internet like [this one](https://github.com/porsager/postgres/issues/62).

IMHO, this could be improved using a parameter `PG_SSL_REJECT_UNAUTHORIZED`. We could set it to true or false depending on the use case. If you think we should do it that way, please add that to the PR. If not, it is working!

Reply:
Forgot to share the error I saw
```
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error [ERR_TLS_CERT_ALTNAME_INVALID]: Hostname/IP does not match certificate's altnames: Host: postgresql.test-instance-6.parabol.local. is not cert's CN: parabol-proving-ground:test-instance-6-postgresql-1b252b94
    at new NodeError (node:internal/errors:372:5)
    at Object.checkServerIdentity (node:tls:346:12)
    at TLSSocket.onConnectSecure (node:_tls_wrap:1542:27)
    at TLSSocket.emit (node:events:527:28)
    at TLSSocket.emit (node:domain:475:12)
    at TLSSocket._finishInit (node:_tls_wrap:946:8)
    at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:727:12) {
  reason: "Host: postgresql.test-instance-6.parabol.local. is not cert's CN: parabol-proving-ground:test-instance-6-postgresql-1b252b94",
  host: 'postgresql.test-instance-6.parabol.local',

IT GOES AND GOES with more stuff that are less relevant
```

Reply:
-1 I don't know. To me it seems to be a setup issue on our side and I would rather fix that than disabling this check. Basing this solely on this comment: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/issues/1735#issuecomment-620667976